taskq/consumer.py

```diff
@@ -63,12 +63,17 @@ def create_scheduled_tasks(self):
         """Register new tasks for each scheduled (recurring) tasks defined in
         the project settings.
         """
+        due_tasks = self._scheduler.due_tasks
+
+        if not len(due_tasks):
```

Comment:
Pas besoin de `len()`, pytlint gueule pour ça. `if not due_tasks` est suffisant.